.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# The bigtable-dpe team is the default owner for anything not
+# explicitly taken by someone else.
+*                               @GoogleCloudPlatform/bigtable-dpe
```

Comment:
please add me as well

Reply:
I need to make googleapis@ versions of the teams per Jeff's comment below. Will amend this and just add you to both teams.

Reply:
Added you to both teams.